packages/manager/modules/manager-pci-common/package.json

```diff
@@ -58,13 +83,10 @@
     "@ovhcloud/ods-components": "^17.2.2",
     "@ovhcloud/ods-theme-blue-jeans": "^17.2.2",
     "@tanstack/react-query": "^5.51.21",
-    "@tanstack/react-query-devtools": "^5.51.21",
-    "@tanstack/react-table": "^8.20.1",
+    "i18next": "^23.8.2",
```

Comment:
This library appears twice with the same version

Reply:
@SimonChaumet
Yes, because `devDependencies` are not bundled, so we need it both here and in `peerDependencies`.
By doing this, we prevent from having a huge bundle of code